### ISSUE TITLE
Use Export and Import

### ISSUE THREAD

**Issue by sulaimonaa** [Aqurds/t-d-l#8]
Great work on this project 👍 However, I have a suggestion: The project follows the DRY principle but all the functions are in the index.js I think you can have some of the functions in an external file, export and import them in the index.js file as part of the project required.

well done 💯 👍 😄

**Comment by Aqurds** [comment]
I agree with the suggested approach @sulaimonaa. I will implement this suggestion soon. Thanks for pointing this out :+1:

**Comment by melybee-07** [comment]
good job so far!! I know you have made a lot of effort  :
congrats on following DRY (Don't Repeat Yourself), KISS (Keep It Simple, Stupid), YAGNI (You Ain't Gonna Need It), and HTML and CSS Best Practices, however, the code don't follow the modularization by separating different functionalities into different files and exporting/importing the necessary functions for JS best practice 
happy coding!👌

**Comment by Aqurds** [comment]
Thanks @melybee-07. I will outsource the index.js file to make it more modularized:+1:
